### PR TITLE
Add blog

### PR DESCRIPTION
**Pull request by lgimeneza**

### CODE REVIEW

**Comment by albanelia0** [comment]
Very well! All perfect!!

File: staff/lgimeneza/blog/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Skylab Blog</title>
```
Comment (agandia9) [inline staff/lgimeneza/blog/index.html:7]: ico here....

File: staff/lgimeneza/blog/index.html [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                <li><a href="/js/">JavaScript</a></li>
+                <li><a href="/jquery/">jQuery</a></li>
+            </ul>
+        </nav>
+    </aside>
+    <main>
+        <article>
+            <header>
+                <h3>Título del tercer post</h3>
+                <time>11 Abril 2018 14:00</time>, author
```
Comment (agandia9) [inline staff/lgimeneza/blog/index.html]: author scapes from any tag...

File: staff/lgimeneza/blog/index.html (hunk 1 above)
Comment (lgimeneza) [reply staff/lgimeneza/blog/index.html:7]: Done!

Comment (lgimeneza) [reply staff/lgimeneza/blog/index.html:7]: done!

**Pull request merged**